### ISSUE TITLE
Reordering connections

### ISSUE THREAD

**Issue by GoogleCodeExporter** [potatomushclient/potato#12]
```
By Mercutio:

I don't know how posible this is. However, it might be a good thing to
allow people to 'move left' and 'move right' the tabs that link to their
screens. It is a thing I've seen many clients lack, yet many people need.
So, maybe you can be the first. (chuckle) (Note that the CTLR-(1 2 3 4 5 6
7 8 9 0) shortcuts will change with that too?)
```

Original issue reported on code.google.com by `mike@keyboardzombie.com` on 6 Jul 2009 at 9:53
- Blocked on: #185
- Merged into: #240
